docs/docs/02-concepts/03-platforms.md

```diff
@@ -0,0 +1,235 @@
+
+### Specifying Multiple Platforms
+
+You may have noticed that the `--platform` option accepts multiple arguments. This means you can specify multiple platforms to compile your application to. For example, if you wanted to compile your application using multiple platforms
+
+```sh
+wing compile app.main.w --platform tf-aws platform-foo platform-bar
+```
+The order in which platforms are evaluated is important. 
+
+The first platform in the list is the primary platform, it is responsible for providing the Wing compiler with the base App that will be used to determine how resources are created, as well it will also lay the ground work for what model the rest of the platforms will need to be compatible with.
+
+### Provisioning Engines
+
+Provisioning is the process of setting up and creating infrastructure, and the provisioning engine is the driver behind this deployment. Common engines used in the Wing compilation process include Terraform and AWS CDK, with support for more planned ([tracking issue](https://github.com/winglang/wing/issues/2066)).
+
+Understanding the differences between provisioning engines will help as we dive deeper into the additional concepts of the platform provider system.
+
+### Platform Models
+
+Platform models are the combination of a provisioning engine and a cloud environment. For example, `tf-aws` is a platform model that uses Terraform as the provisioning engine and AWS as the cloud environment. Similarly, `awscdk` is a platform model that uses AWS CDK as the provisioning engine and AWS as the cloud environment.
+
+It is worth noting that the platform names are not guaranteed to match their models, we will see this more as we delve into the idea of Custom Platforms below.
+
+Though not currently implemented, the platform model system is designed with extensibility in mind, as it will be used to determine compatibility between different platforms ([tracking issue](https://github.com/winglang/wing/issues/1474))
+
+#### Model-specific code
+
+There might be times when you need to write code that is specific to a particular platform model. For example, you may want to activate a verbose logging service only when testing locally to save on cloud log storage costs.
+
+With the Wing `util` library, you can access environment variables. The `WING_MODEL` environment variable contains the current platform model as it's value, which you can use to conditionally run model-specific code. See the example below:
+
+```js playground
+bring cloud;
+bring util;
+
+let invocationCounter = new cloud.Counter();
+let queue = new cloud.Queue();
+
+queue.setConsumer(inflight (msg: str) => {
+  invocationCounter.inc();
+});
+
+new cloud.Function(inflight ()=> { 
+  // push a message to queue
+  queue.push("m");
+  // sleep according to model 
+  if util.env("WING_MODEL") == "sim" {
```

Comment:
I am thinking that perhaps we should use the term "target" instead of "model". WDYT? It would work and might feel a bit more natural, especially in this context.

Reply:
I feel like there might be some more exploring that needs done here, I agree in this context "target" sounds better, then in others I feel like "model" is a more accurate term. That said, Im in favor of going back to target for now